docs/howto/index.md

```diff
@@ -6,18 +6,20 @@ specific tasks in {{config.extra.brand}} and the services we support.
 There are several categories of How-To guides, and they tend to be
 focused on a specific cloud technology.
 
-* **OpenStack CLI/API** How-Tos cover tasks that you can accomplish
+* **Kubernetes** How-Tos covering how you can create and manage your Kubernetes deployments using {{config.extra.gui}}.
```

Comment:
I *think* it's wise to keep OpenStack at the top, followed by object storage and then Kubernetes, rather than moving Kubernetes to the top of the list. Is there a particular reason why you are suggesting to change the order?

Reply:
My initial reason was to have the list items on this page reflect the order of the pages in the sidebar, which sorts alphabetically by default. That is:
* Kubernetes
* Object storage
* OpenStack

This is probably not really important, just some visual thing I noticed. I guess better way to organise the sidebar entries is to use .page file and _nav_ property and adjust that to the page content, not other way around.
Let me know what do you think about it. And if it were to be included, then if it's better to create a separate, small PR with that fix.

Reply:
Ah, yes, that's a perfectly fair point. I stand corrected. Leave the order as is, please.